### PR TITLE
refactor/796

### LINKED ISSUE #796: ENGM Needs an update
It's not even running at the moment so it's desperately asking to be brought back.

### PR DESCRIPTION
**Pull request by Fechulo**
Resolves #796 

The purpose of this pull request is to update ENGM and continue to figure out how git works

### Completion Checklist
- [x] terrain file
- [x] polygonal airspace boundary
- [x] all SIDs
- [x] all STARs
- [x] information to validate traffic levels
- [x] base video map
- [x] no console errors or warnings
- [x] airport guide
---

According to Wikipedia, ENGM has 237,618 yearly operations. Traffic level should fall between 27.1acph and 81.4acph. Currently, there's a total of 28acph 👍

### CODE REVIEW

**Comment by erikquinn** [comment]
Removing request for my review due to `WIP` flag for now, re-request when needed! 😄

File: assets/airlines/nax.json [hunk 1]
```diff
@@ -4,21 +4,24 @@
...
-            ["B789", 2]
-        ],
-        "long": [
-            ["B788", 8 ],
-            ["B789", 2]
+            ["B789", 18]
         ],
         "short": [
-            ["B738", 90]
+            ["B738", 117]
```
Comment (indianbhaji) [inline assets/airlines/nax.json]: No 737 MAXs?

File: assets/airlines/ibk.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        "callsignFormats": [
+            "####",
+            "##@@",
+            "###",
+            "#@@"
+        ]
+    },
+    "fleets": {
+        "default": [
+            ["B738", 70]
```
Comment (indianbhaji) [inline assets/airlines/ibk.json]: No 737 MAXs?

File: assets/airports/engm.json [hunk 3]
```diff
@@ -13,825 +13,1920 @@
...
+                ["N60.697421000", "E10.522672100"],
+                ["N60.698052500", "E10.521450800"],
+                ["N60.698714200", "E10.520298800"],
+                ["N60.699404500", "E10.519219100"],
+                ["N60.700121400", "E10.518214600"],
+                ["N60.700862900", "E10.517288200"],
+                ["N60.701627100", "E10.516442100"],
+                ["N60.702411800", "E10.515678800"],
+                ["N60.703214800", "E10.515000300"],
+                ["N60.704034000", "E10.514408500"
```
Comment (indianbhaji) [inline assets/airports/engm.json]: Should this * be here? It's not a transition, is it?

Comment (indianbhaji) [inline assets/airports/engm.json]: Extra line

Comment (indianbhaji) [inline assets/airports/engm.json]: No speed restriction here at `GM440`, it appears that this restriction only applies to NUVSA9B.

Comment (indianbhaji) [inline assets/airports/engm.json]: No restriction here at `GM440`

Comment (indianbhaji) [inline assets/airports/engm.json]: Flyover

Comment (indianbhaji) [reply assets/airports/engm.json]: (at least it looks like it)

Comment (indianbhaji) [inline assets/airports/engm.json]: Flyover

Comment (indianbhaji) [inline assets/airports/engm.json]: Flyover

Comment (indianbhaji) [inline assets/airports/engm.json]: Hold at VALPU

Comment (indianbhaji) [inline assets/airports/engm.json]: Hold at BAVAD

Comment (indianbhaji) [inline assets/airports/engm.json]: Hold at VALPU

Comment (indianbhaji) [inline assets/airports/engm.json]: Hold at BAVAD

Comment (indianbhaji) [inline assets/airports/engm.json]: Hold at INSUV

Comment (indianbhaji) [inline assets/airports/engm.json]: Hold at TITLA

Comment (indianbhaji) [inline assets/airports/engm.json]: Hold at INSUV

Comment (indianbhaji) [inline assets/airports/engm.json]: Hold at TITLA

Comment (indianbhaji) [inline assets/airports/engm.json]: Hold at INSUV

Comment (indianbhaji) [inline assets/airports/engm.json]: `A120-` on GM447
Hold at TITLA

Comment (indianbhaji) [inline assets/airports/engm.json]: Hold at VALPU

Comment (indianbhaji) [inline assets/airports/engm.json]: Hold at BAVAD

File: documentation/airport-guides/engm.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
+### About this facility
+Oslo TMA, Røyken, Norway
+237,618 annual operations in 2016 (Reported by Avinor)
+
+The Oslo TMA is one of three sectors of the Oslo Kontrollsentral (ATCC), responsible for all airspace in Eastern Norway. The Oslo ATCC is located 30km from Oslo and 65km Southwest of Oslo Gardemoen Airport (ENGM).
+
+### Airspace
+the Oslo TMA consists of Class C airspace (equivalent to American class B airspace) which covers a large area on top of Akershus County and the Oslo metropolitan area up to FL215. it also includes a small shelf to the South with a floor of FL115; below this, the airspace is ceded to the Tarris TMA.
```
Comment (indianbhaji) [inline documentation/airport-guides/engm.md]: Capital T on 'The'
Capital I on 'It'

File: documentation/airport-guides/engm.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+### Instrument Procedures
+The procedures are quite modern for a European airport. All SID's and STAR's are RNAV and include both altitude and speed restrictions. Oslo was also the first airport in the world to adopt a point merge system, minimizing conflicts between departing and arriving aircraft as well as reducing fuel consumption. Once within the point merge arches, aircraft can be given shortucts or vectors to join the ILS approach.
```
Comment (indianbhaji) [inline documentation/airport-guides/engm.md]: SIDs and STARs
Are descent clearances given by ATC?

File: documentation/airport-guides/engm.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+W - West Radar  (unavailable)
+```
+
+### Airports of Interest
+
+#### ENGM (Primary Airport)
+Oslo Gardermoen Airport  
+237,618 annual operations
+
+located 35km Northeast of Oslo, Gardermoen is the main international airport in Norway and the second busiest in Scandinavia. The airport has two parallel runways (01-19) used simultaneously; Usually 01L is used for departures while 01R is used for arrivals.
```
Comment (indianbhaji) [inline documentation/airport-guides/engm.md]: Capital L
Lowercase u on 'usually'

File: assets/airports/engm.json [hunk 7]
```diff
@@ -13,825 +13,1919 @@
...
```
Comment (indianbhaji) [inline assets/airports/engm.json]: What about these `*`s on the STARs? (1034, draw sections on STARs)

**Pull request merged**